java/cuvs-java/src/main/java22/com/nvidia/cuvs/internal/common/Util.java

```diff
@@ -155,10 +145,10 @@ public static List<GPUInfo> compatibleGPUs() throws Throwable {
   public static List<GPUInfo> compatibleGPUs(double minComputeCapability, int minDeviceMemoryMB)
       throws Throwable {
     List<GPUInfo> compatibleGPUs = new ArrayList<GPUInfo>();
-    double minDeviceMemoryB = Math.pow(2, 20) * minDeviceMemoryMB;
+    double minDeviceMemoryB = 1048576.0f * minDeviceMemoryMB;
```

Comment:
Nit: Not sure about using that magic number.  Should we either consider writing this as:

```suggestion
    double minDeviceMemoryB = 1024f * 1024 * minDeviceMemoryMB;
```

Reply:
Don't see why not, it's clearer. I'll make the change.